lib/cli.js

```diff
@@ -55,7 +55,7 @@ async function start() {
   let table = new Table({
     head: ['Spec', 'Time', 'Tests', 'Passing', 'Failing', 'Pending'],
     style: { head: ['blue'] },
-    colWidths: [50, 8, 7, 9, 9, 9]
+    colWidths: [100, 10, 10, 10, 10, 10]
```

Comment:
Thanks!A fixed width could not work for everyone, and people using small screens would end up seeing a messed up table. I think I found a better solution by making the width flexible based on the spec name length: https://github.com/tnicola/cypress-parallel/blob/f758d4533ee3df353641cb529c6ae0ab4bfa52e4/lib/cli.js#L58